lib/wda/webdriveragent.js

```diff
@@ -96,7 +96,11 @@ class WebDriverAgent {
     }
   }
 
-  async isRunning () {
```

Comment:
I found some `isRunning` in other files. They are not WDA thing so the change affect only https://github.com/appium/appium-xcuitest-driver/pull/709/files#diff-f64f06d264dffd32eb8d87f1b3095989R420

```
lib/commands/performance.js:  if (!proc.isRunning) {
lib/device-log/ios-log.js:      if (!await this.sim.isRunning()) {
lib/device-log/ios-log.js:    return !!(this.proc && this.proc.isRunning);
lib/simulator-management.js:    if (await device.isRunning()) {
test/functional/driver/driver-e2e-specs.js:        (await sim.isRunning()).should.be.true;
test/functional/driver/driver-e2e-specs.js:        (await sim.isRunning()).should.be.false;
test/functional/driver/driver-e2e-specs.js:        (await sim.isRunning()).should.be.true;
test/functional/driver/driver-e2e-specs.js:        (await sim.isRunning()).should.be.true;
test/functional/driver/driver-e2e-specs.js:        (await sim.isRunning()).should.be.true;
test/unit/driver-specs.js:            isRunning () {
```

Reply:
you can keep both methods, but you can still extract the common stuff, which retrieves status dictionary